src/test/regress/expected/shared_connection_stats.out

```diff
@@ -531,8 +531,8 @@ BEGIN;
 		hostname, port;
  connection_count_to_node
 ---------------------------------------------------------------------
-                        3
-                        3
+                        8
+                        4
```

Comment:
how reliable are these numbers?

Reply:
Thanks for spotting, probably not very reliable,

 I should use `> 3` and use some larger set to COPY into so that we make sure that we hit at least 4 shards on every worker. That's what we want to show anyway. (max_shared_pool_size == 3)